README.md

```diff
@@ -31,6 +31,13 @@ Install the latest release:
 pip install keras-tuner --upgrade
 ```
 
+Install _optional_ dependencies for the `Bayesian` tuner:
+
+```
+pip install keras-tuner[bayesian]
+```
+
+
```

Comment:
We would like to keep the instructions in README.md simple and only inform the users in the error messages when they use the `BayesianOptimization`.
So please remove the changes here.

Reply:
Thx @haifeng-jin, I like your idea 😊

Reply:
✅